### DISCUSSION TITLE
Kompatibilität Daikin Altherma 3R MMI

### DISCUSSION THREAD

**Discussion by Enrico5444** [Trunks1982/Daikin-Rotex-HPSU-CAN#34] [Q&A]
Hallo Trunks1982, 

ich finde du hast hier ein schönes und interessantes Projekt aufgebaut. Ich nutze derzeit ESP Altherma in Kombination mit der Onecta Developer API (welche ich als nicht langfristig verfügbar sehe). Lieder kann ich hierüber nicht die WW temp verändern und musste mir aufwendig eine Automation für die Heizkurve sowie die Umschaltung zwischen kühlen und heizen basteln. 
Voller Euphorie habe ich mich auch gleich an die Umsetzung deines Projekts gemacht und wollte meine Altherma 3RF via Can-Bus in Home Assistent einbinden. Naja wer lesen kann ist klar im Vorteil, habe nach dem Einrichten des ESP gemerkt, dass es sich bei deinem Projekt um die HPSU mit RoCon Steuerung handelt die anscheint Can nutzt und die Altherma 3RF kein J13 Steckplatz besitzt.

Da es mich nicht losgelassen hat, dass die Altherma 3R keinen J13 Stecker der Can-Bus besitz habe ich etwas weiter geforscht und bin darauf gestoßen, dass die Modelle 3RF auch eine Can-Bus Schnittstelle haben. 
Das Modul EKMIKPOAF BizoneKitMMI kommuniziert an der MMi Platine am Steckplatz X3 dann auch über Can-Bus. 

Modul EKMIKPOAF BizoneKitMMI
![image](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/129221890/f37c3ecb-319a-4535-9841-ac253ff83d4f)

MMI Platine
![image](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/129221890/a54254ab-5f80-4710-8669-4b0b99c443de)

Jetzt wäre die Frage sind wäre dien Projekt evtl. doch Kompatibel mit den Altherma 3RF Geräten? 
Wo / Wie könnte man die Can Daten mit deinen evtl. abgleichen? 
Würdest du ggf. die Kompatibilität mit in dein Projekt mit aufnehmen?

Beim testen oder weiterer Recherche kann ich gern unterstützen.

Schöne Grüße

**Comment by Trunks1982** [comment]
Ob die kompatibel sind kann ich dir nicht sagen aber ich würde mein projekt einfach mal anschliessen und gucken was für werte rein kommen.Dann kann man ja mal weiter sehn.

**Reply by Enrico5444** [reply]
leider bekomme ich keine werte rein... 
Ich weiß jetzt allerdings nicht, ob die Daten mit der HPSU und der Altherma 3RF übereinstimmen oder ob die Schnittstelle X3 gar keine Daten zur Verfügung stellt.

Kannst du mir evtl. sagen wo du deine Can daten her hast?

**Reply by Trunks1982** [reply]
Fhem, iobroker.Was du versuchen könntest das logging verbose anzumachen und mal an der Heizung was verstellen und gucken ob can daten reinkommen.

**Reply by Enrico5444** [reply]
ich nutze Home Assistent.
![image](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/129221890/16b7a2be-c5eb-4aac-a190-9c89fc39ef27)

**Reply by Trunks1982** [reply]
Du hast ja gefragt wo ich die can daten her habe.
